server/app/services/export/PdfExporter.java

```diff
@@ -369,6 +370,22 @@ private void renderProgramBlock(
         document.add(
             text(question.getQuestionHelpText().getDefault(), PARAGRAPH_FONT, indentationLevel));
       }
+
+      // Adds a line describing whether the question is optional or not
+      Optional<ProgramQuestionDefinition> programQuestionDefinition =
+          block.programQuestionDefinitions().stream()
+              .filter(pqd -> pqd.id() == question.getId())
+              .findFirst();
+      if (programQuestionDefinition.isPresent()) {
+        document.add(
+            text(
+                programQuestionDefinition.get().optional()
+                    ? "Required Question"
+                    : "Optional Question",
```

Comment:
(blocking) These are backwards

Reply:
Oops, idk why i thought they were supposed to be flipped. should be fixed
[comprehensive-sample-program-2024-07-24T14_59_44.718985217.pdf](https://github.com/user-attachments/files/16367879/comprehensive-sample-program-2024-07-24T14_59_44.718985217.pdf)